examples/GLTextItem.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+"""
+Simple examples demonstrating the use of GLTextItem.
+
+"""
+
```

Comment:
please add this line as the first import statement

```python
import initExample ## Add path to library (just for examples; you do not need this)
```

We do this to support examples running when pyqtgraph hasn't actually been installed via `pip`

Reply:
I fixed in [
b98244e](https://github.com/pyqtgraph/pyqtgraph/pull/1776/commits/b98244e7d34ce368d6d49af76545daf2d4f0ade1).

Reply:
👍🏻